lib/data/wrapper.js

```diff
@@ -59,6 +59,23 @@ if (config.backends.data === 'mem') {
  */
 const MAX_RETRY = 2;
 
+// This check is done because on a put, complete mpu or copy request to AWS,
+// if the object already exists on AWS, the existing object should not be
+// deleted, which is the functionality for all other backends
+function _shouldSkipDelete(locations, requestMethod, newObjDataStoreName) {
+    const skipBackend = externalBackends;
```

Comment:
what does this do?

Reply:
Nothing much, it's leftover from the original check. I removed it